packages/angular_devkit/build_angular/src/utils/run-module-as-observable-fork.ts

```diff
@@ -11,7 +11,7 @@ import { resolve } from 'path';
 import { Observable } from 'rxjs';
 const treeKill = require('tree-kill');
 
-
+// TODO: refactor out the fork seprate from the require module part
```

Comment:
Just a reminder split up this logic since i'd like to use the fork part of it instead of rolling my own